### PR TITLE
suggest to end PHP variable assignements with semicolon

### PR DESCRIPTION
**Pull request by jnweiger**
Not sure, if the config.php files also work without semicolon (;) or with trialing comma (,) --

backports to at least 10.9

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/external_storage/s3_compatible_object_storage_as_primary.adoc [hunk 1]
```diff
@@ -60,7 +60,7 @@ $CONFIG = [
             ...
         ],
     ],
-]
+];
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/files/external_storage/s3_compatible_object_storage_as_primary.adoc:63]:
```suggestion
],
```

File: modules/admin_manual/pages/configuration/files/external_storage/s3_compatible_object_storage_as_primary.adoc [hunk 2]
```diff
@@ -92,7 +92,7 @@ $CONFIG = [
             ],
         ],
     ],
-],
+];
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/files/external_storage/s3_compatible_object_storage_as_primary.adoc:95]:
```suggestion
],
```

**Comment by mmattel** [comment]
> Not sure, if the config.php files also work without semicolon (;) or with trialing comma (,) --

We use commas. Your changes have been adopted therefore.
Thanks for catching :smiley:

**Review by mmattel — APPROVED** [review]

**Comment by jnweiger** [comment]
> > Not sure, if the config.php files also work without semicolon (;) or with trialing comma (,) --
> 
> We use commas. Your changes have been adopted therefore. Thanks for catching smiley

A commas is not a legal syntax to end a php assignement with. -> #4288

**Pull request merged**